package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/bulk-api-adapter",
-  "version": "8.3.1-snapshot",
+  "version": "8.3.1",
```

Comment:
this should still be snapshot as there are no tests

Reply:
Package version back to being a "-snapshot"